packages/core/test/final.test.ts

```diff
@@ -113,7 +113,7 @@ describe('final states', () => {
             reveal: {
               type: 'final',
               output: {
-                secret: () => 'the secret'
+                secret: 'the secret'
```

Comment:
This test no longer tests what it should (`'should call data expressions on nested final nodes'`). To fix this we could make the whole `output` an expression here